### PR TITLE
DP-2433: exports docstrings

### PR DESCRIPTION
**Pull request by Mara3l**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: docs/content/en/docs/workspace-content/dashboard-export/export_tabular_by_insight_id.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+linkTitle: "export_tabular_by_insight_id"
+weight: 110
+superheading: "export."
+---
+
+``export_tabular_by_insight_id(
+        workspace_id: str,
+        insight_id: str,
+        file_format: str,
+        use_labels: bool = True,
```
Comment (hkad98) [inline docs/content/en/docs/workspace-content/dashboard-export/export_tabular_by_insight_id.md:12]: This was removed. Please adapt to the change.

File: docs/content/en/docs/workspace-content/dashboard-export/export_tabular_by_insight_id.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        file_name: Optional[str] = None,
+        settings: Optional[ExportSettings] = None,
+        custom_override: Optional[ExportCustomOverride] = None,
```
Comment (hkad98) [inline docs/content/en/docs/workspace-content/dashboard-export/export_tabular_by_insight_id.md:15]: This was removed. Please adapt to the change.

File: docs/content/en/docs/workspace-content/dashboard-export/export_pdf.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+{{% parameters-block title="Returns" None="true"%}}
+{{% /parameters-block %}}
+
+
+## Example
+
+```python
+
+## TODO: Hkad98
```
Comment (hkad98) [inline docs/content/en/docs/workspace-content/dashboard-export/export_pdf.md:52]:
```python
host = "https://www.example.com"
token = "<your_personal_access_token>"
sdk = GoodDataSdk.create(host, token)


sdk.export.export_pdf(workspace_id="demo",  dashboard_id="campaign", file_name="test")
```

File: docs/content/en/docs/workspace-content/dashboard-export/export_tabular.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+{{% parameters-block title="Returns" None="true"%}}
+{{% /parameters-block %}}
+
+
+## Example
+
+```python
+
+## TODO: Hkad98
```
Comment (hkad98) [inline docs/content/en/docs/workspace-content/dashboard-export/export_tabular.md:52]:
```python
host = "https://www.example.com"
token = "<your_personal_access_token>"
sdk = GoodDataSdk.create(host, token)

workspace_id = "demo"
exec_def = ExecutionDefinition(
        attributes=[Attribute(local_id="region", label="region"), Attribute(local_id="state", label="state")],
        metrics=[
            SimpleMetric(local_id="price", item=ObjId(id="price", type="fact")),
            SimpleMetric(local_id="order_amount", item=ObjId(id="order_amount", type="metric")),
        ],
        filters=[],
        dimensions=[["state", "region"], ["measureGroup"]],
    )
execution_result=sdk.compute.for_exec_def(workspace_id, exec_def).result_id
export_request = ExportRequest(
        format="CSV",
        execution_result=execution_result,
        file_name="my_file",
        custom_override=ExportCustomOverride(
            labels={"region": ExportCustomLabel(title="Custom Title Region")},
            metrics={
                "price": ExportCustomMetric(title="Sum Of Price", format=""),
                "order_amount": ExportCustomMetric(title="Order Amount Metric", format="#,##0.00"),
            },
        ),
    )


sdk.export.export_tabular(workspace_id, export_request, _exports_dir)
```

File: docs/content/en/docs/workspace-content/dashboard-export/export_tabular_by_insight_id.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+{{% parameters-block title="Returns" None="true" %}}
+{{% /parameters-block %}}
+
+
+## Example
+
+```python
+
+## TODO: Hkad98
```
Comment (hkad98) [inline docs/content/en/docs/workspace-content/dashboard-export/export_tabular_by_insight_id.md:66]:
```python
host = "https://www.example.com"
token = "<your_personal_access_token>"
sdk = GoodDataSdk.create(host, token)

sdk.export.export_tabular_by_insight_id(
        workspace_id="demo", insight_id="campaign_spend", file_format="CSV")

```

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Pull request closed without merging**